types/custom-functions-runtime/index.d.ts

```diff
@@ -75,6 +75,16 @@ declare namespace CustomFunctions {
          * [Api set: CustomFunctionsRuntime 1.3]
          */
         parameterAddresses?: string[];
+
+        /**
+         * Indicates whether the function is invoked as part of the formula value preview.
+         * On Desktop this is `true` if the function is invoked is to preview the formula value; otherwise it is `false`.
+         * On Mac, this is always `false`.
+         * 
```

Comment:
Could we explain that it can't be set by an add-in? Maybe like this:
```suggestion
         * `isInValuePreview` is read-only and can't be set by a custom functions add-in. Use the Excel UI to adjust this value. 
         *
```

Reply:
There is no real way to adjust this value. But yes, I will mention that it is set by Excel and can't be set by an add-in.